js/packages/web/package.json

```diff
@@ -5,18 +5,14 @@
     "@ant-design/icons": "^4.4.0",
     "@babel/preset-typescript": "^7.12.13",
     "@cloudflare/stream-react": "^1.1.0",
-    "@oyster/common": "0.0.1",
+    "@oyster/common": "0.0.2",
```

Comment:
Once https://github.com/solana-labs/oyster/pull/180 is merged it seems like it might be possible to refactor and remove the local common package.